runtime/onert/backend/cpu/Tensor.h

```diff
@@ -29,8 +29,11 @@ namespace cpu
 
 using Tensor = cpu_common::Tensor;
 
-// Tensor which has data from external. To support this, assume below things
-// no padding, always NHWC layout, constant tensor and not dynamic
+/**
+ * @brief Class that uses data from external. This class uses the data from external
```

Comment:
how about writing `external` to the following ? To me, `external` is difficult to understand.
```
 * @brief Class that uses data from external memory that is not managed by a backend. For example, ExternalTensor's data pointer can point to mmapped area of tflite or circle file.
```

Reply:
That's very good description. I'll fix like it. Thanks.

Reply:
I think "mmaped area" is too narrow. As this class is a wrapper of `ir::Data`, not just `ir::MMapedData`. The description should be more general.

Reply:
@wateret 
> I think "mmaped area" is too narrow. As this class is a wrapper of ir::Data, not just ir::MMapedData. The description should be more general.
Oh yes, you're right. The tensor could be all of types' ir::Data. I'll fix it soon.